spec/lib/dttp/params/contact_spec.rb

```diff
@@ -166,6 +166,47 @@ module Params
             end
           end
         end
+
+        context "nationality information" do
+          let(:expected_nationality) { create(:nationality, name: name) }
+          let(:other_nationality) { create(:nationality, name: "other") }
```

Comment:
```suggestion
          let(:expected_nationality) { create(:nationality, :irish }
          let(:expected_nationality) { create(:nationality, :british }
          let(:other_nationality) { create(:nationality, :other) }
```

fussy stuff, can we build up our factories?

Reply:
I put them into traits but only used `other` here https://github.com/DFE-Digital/register-trainee-teachers/pull/564/commits/00eba96ddd2d717211ed74abda9f91646283183d